### PR TITLE
Add package germanium

### LINKED ISSUE #6505: Package request: germanium
https://github.com/matsuyoshi30/germanium

It's just a Go package so it should be an easy addition.

If anyone else is interested the arm64 binaries from the releases works perfectly out of the box on termux.

### PR DESCRIPTION
**Pull request by ravener**
Resolves #6505

First time adding a package, hopefully did not mess up anything.

### CODE REVIEW

File: packages/germanium/build.sh [hunk 1]
```diff
@@ -0,0 +1,18 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/matsuyoshi30/germanium
+TERMUX_PKG_DESCRIPTION="Germanium is an alternative to Carbon and Silicon implemented in Go."
```
Comment (buttaface) [inline packages/germanium/build.sh:2]: Use the project's own description: `Generate image from source code`.

**Comment by ravener** [comment]
@buttaface done

File: packages/germanium/build.sh [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+TERMUX_PKG_LICENSE="MIT"
+TERMUX_PKG_MAINTAINER="Raven Ravener <ravener.anime@gmail.com>"
+TERMUX_PKG_VERSION=0.3.0
+TERMUX_PKG_SRCURL=https://github.com/matsuyoshi30/germanium/archive/v$TERMUX_PKG_VERSION.tar.gz
+TERMUX_PKG_SHA256=44fc26bb263f409363618ab091b7ca877f11870edbff436efc5fe2d4e950beda
+ 
+termux_step_make() {
+	termux_setup_golang
+ 
+	cd $TERMUX_PKG_SRCDIR
```
Comment (Grimler91) [inline packages/germanium/build.sh:12]: It should be possible to set `TERMUX_PKG_BUILD_IN_SRC=true` and not have to `cd $TERMUX_PKG_SRCDIR`

File: packages/germanium/build.sh [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+	go build
+}
+ 
+termux_step_make_install() {
+	install -Dm700 -t "$TERMUX_PREFIX"/bin $TERMUX_PKG_SRCDIR/germanium
```
Comment (Grimler91) [inline packages/germanium/build.sh:17]: With `TERMUX_PKG_BUILD_IN_SRC=true` you can also replace `TERMUX_PKG_SRCDIR/germanium`->`germanium`

**Review by Grimler91 — COMMENTED** [review]
Looks good! See the minor comments I had

**Comment by ravener** [comment]
@Grimler91 fixed. Thanks.

**Pull request merged**